charts/kube-master/templates/dex.yaml

```diff
@@ -0,0 +1,264 @@
+{{ if and .Values.dashboard .Values.dashboard.enabled }}
```

Comment:
Then the flag options will be like:
- **dex** (true/false) 
- if **dex**  is true
  - **dashboard and dashboard proxy** (true/false)

I guess it will add another field in `KlusterSpec` or do you suggest any other option?

Reply:
It is implemented by: https://github.com/sapcc/kubernikus/pull/464/commits/1a4761706b52e0e6c7d145206b911197dc7ad45a

Manually tested configurations:
Dex: true
- Dashboard: false  --> Only dex is installed, CLI access with OIDC login works.
- Dashboard: true --> Dashboard and proxy is additionally installed.